src/App.css

```diff
@@ -1,8 +1,15 @@
+* {
```

Comment:
Multiplication functionality is not working as expected. Kindly see the error below from console.
![errors](https://user-images.githubusercontent.com/35504149/148601265-78323d90-5c45-4f19-8d30-3b89290cca7a.png)